src/common/entity/compute_state_display.ts

```diff
@@ -55,6 +55,12 @@ export const computeStateDisplay = (
     return formatDateTime(date, language);
   }
 
+  if (domain === "humidifier") {
+    if (stateObj.state === "on" && stateObj.attributes.humidity) {
+      return `${stateObj.attributes.humidity} %`;
```

Comment:
This is a bit weird, we should show the state instead if an attribute?

Reply:
The idea is to present to the user the most useful information on the glance card. The state is either on or off, and when it is on it is much more informative to show the target humidity than the state. And when it's off, it doesn't matter so much what was the target humidity. Also, the on/off state is represented by the icon color (and I also plan to use the mdi:air-humidifier-off icon which is not yet available).
Please check the screenshot in the 'Proposed change' section for the example.

Reply:
This is by the way in line with how some of the humidifiers represent their user interface. Something like this:
![humidifier_control_panel](https://user-images.githubusercontent.com/2741408/85040937-01195380-b192-11ea-9aed-e0acf77bb798.png)

Reply:
Percentage signs should not have a space between number and `%`

Reply:
Well, currently all units have a space between it and a number. For example check line 19 of the same file, or even the weather card and the humidity slider in the more-info dialog for the thermostat card.
Do you want me to change that as well?

Reply:
Yes but not in this PR.

Reply:
Done